### PR TITLE
PRIME-2794 Change vertical dot menu to buttons

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html [hunk 1]
```diff
@@ -174,25 +174,29 @@
...
+    <div class="mat-header-button-group">
+      <button mat-button
+              class="title_button"
+              type="button"
+              color="primary"
+              title="View/Update Organization Information"
+              [disabled]="isPendingTransfer()"
               (click)="viewOrganization(organization)">
-        <span>View/Update Organization</span>
+        <mat-icon>folder</mat-icon>
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html:185]: Just my subjective opinion, but `description` icon seems better

File: prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html [hunk 2]
```diff
@@ -201,31 +205,34 @@
...
+    <div class="mat-header-button-group">
+      <button mat-button
+              class="title_button"
+              type="button"
+              color="primary"
+              title="View/Update Site Information"
+              [disabled]="isPendingTransfer()"
               (click)="viewSite(organizationId, site)">
-        <span>View/Update Site Information</span>
+        <mat-icon>folder</mat-icon>
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html:221]: Just my subjective opinion, but `description` icon seems better

**Comment by neophyte57** [comment]
Do we need to support mobile device usage?  
image

File: prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html [hunk 3]
```diff
@@ -201,31 +205,34 @@
...
+              class="title_button"
+              type="button"
+              color="primary"
+              title="View/Update Remote Practitioner"
+              *ngIf="site.careSettingCode === CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE"
+              [disabled]="isPendingTransfer() || !site.completed || site.status === SiteStatusType.IN_REVIEW || site.status === SiteStatusType.LOCKED"
               (click)="viewSiteRemoteUsers(organizationId, site)">
-        <span>View/Update Remote Practitioners</span>
+        <mat-icon>folder_shared</mat-icon>
+        <span class="pl-1">Remote Practitioner</span>
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html:233]: IMO, this should be "Remote Practitioner**s**"

**Comment by neophyte57** [comment]
Good catch re: one omission for Locked sites but when Locked, still editable?
image

**Comment by neophyte57** [comment]
Upon clicking the agreement "link" under Downloads:
image

Get this:
image

Should the "Downloads" button be disabled if Pending Transfer?

File: prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html (hunk 2 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html]: Agree. It is updated.

File: prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html (hunk 3 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/site-registration/pages/site-management-page/site-management-page.component.html]: it is updated.

**Comment by bergomi02** [comment]
> Do we need to support mobile device usage? image

Updated the css to hide the text (only showing the icon). I think it looks better.

**Comment by bergomi02** [comment]
> Upon clicking the agreement "link" under Downloads: image
> 
> Get this: image
> 
> Should the "Downloads" button be disabled if Pending Transfer?

It is fixed.

**Comment by bergomi02** [comment]
> Good catch re: one omission for Locked sites but when Locked, still editable? image

It is fixed.

**Review by neophyte57 — APPROVED** [review]
👍 Nice work!

**Pull request merged**